Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -585,5 +585,91 @@ public class GraphLinter: GraphLinting {
             LintableTarget(platform: .macOS, product: .framework),
             LintableTarget(platform: .macOS, product: .staticFramework),
         ],
+
```

Comment:
I might be wrong, but it seems that some items in this list might have been copied over from iOS targets and might not apply to the context of `visionOS` apps. I'd recommend going through them and ensuring they are correct.